iommi/traversable.py

```diff
@@ -116,7 +116,7 @@ def iommi_dunder_path(self) -> str:
         return build_long_path(self).replace('/', '__')
 
 
-    def bind(self, *, parent=None, request=None):
+    def bind(self, *, parent=None, request=None, _name=None):
```

Comment:
Where is this used?

Reply:
https://github.com/TriOptima/iommi/pull/72/files#diff-f9cd0e4040f47dd1fa31ae7b3ab4628674ceb54fa11f3dae335154f3c819cfb6R659
https://github.com/TriOptima/iommi/pull/72/files#diff-f9cd0e4040f47dd1fa31ae7b3ab4628674ceb54fa11f3dae335154f3c819cfb6R697
https://github.com/TriOptima/iommi/pull/72/files#diff-67dfe7b6d19389647a0b3fc4d18c9d1dfbcd9897434d45d1c16938811d4fd8c7R1628
https://github.com/TriOptima/iommi/pull/72/files#diff-f9cd0e4040f47dd1fa31ae7b3ab4628674ceb54fa11f3dae335154f3c819cfb6R659
and some more places...

Reply:
I was thinking of the usage of the parameter. I find no place in any `bind` actually using the parameter. Am I missing something? From what I see we get the _name in the constructor and only modify it in `bind` if we are in the root.